FantasmoSDK.podspec

```diff
@@ -17,7 +17,7 @@ Pod::Spec.new do |s|
   s.author           = { 'krishn' => 'krishn@github.io' }
   s.source           = { :git => 'https://krishn-fantasmo@github.com/fantasmoio/Fantasmo-iOS-SDK.git', :branch => 'master', :tag => s.version.to_s }
 
-  s.ios.deployment_target = '11.0'
+  s.ios.deployment_target = '12.0'
```

Comment:
Let's undo this?

Reply:
I have noticed that SDK had 12 in xcode project settings and just synchronized values.

Can we set 11.3 to avoid additional swift directives for resolving this issue https://take.ms/8D5LH?

Reply:
Oh, I see, then 12 is fine.